src/GovUk.Education.ExploreEducationStatistics.Admin/Services/SignInService.cs

```diff
@@ -184,4 +196,36 @@ await contentDbContext.RequireTransaction(async () =>
             await usersAndRolesDbContext.SaveChangesAsync();
         });
     }
+
+    private async Task<Guid> GetDeletedUserId() =>
+        (await userRepository.FindDeletedUserPlaceholder())!.Id;
```

Comment:
I think I spotted a change where you altered this to return `User` rather than `User?` so the null-overriding operator `!` shouldn't be necessary here.